packages/oui-form-actions/README.md

```diff
@@ -0,0 +1,48 @@
+# Form Actions
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+```html
+<oui-form-actions
+  on-submit="$ctrl.submit()"
+  on-cancel="$ctrl.cancel()">
+</oui-form-actions>
+```
+
+## API
```

Comment:
Api doc should be at the end like the majority of components.

Reply:
oki